webapp/javascript/components/TimelineChart/Tooltip.plugin.tsx

```diff
@@ -115,6 +115,10 @@ const TOOLTIP_WRAPPER_ID = 'explore_tooltip_parent';
           }
         );
 
+        if (!values?.length) {
```

Comment:
Bit of a sidebar, but there's another edge case we don't fully handle:
It happens when we pass a custom tooltip (on that onHoverDisplayTooltip callback), which in turn returns `null`.

Here's an illustration of the problem:
<img width="881" alt="Screen Shot 2022-09-07 at 10 46 08" src="https://user-images.githubusercontent.com/6951209/188894303-be998546-c771-4658-bddb-7d4388aa8ee6.png">